src/scripts/commands/database.ts

```diff
@@ -0,0 +1,205 @@
+    const removeOrgUnitLevelCommand = command({
+        name: "remove-orgunit-level",
+        description: "Create a sql file that removes organisation units below a hierarchy level",
+        args: {
+            level: positional({
+                type: HierarchyLevel,
+                description: "Maximum hierarchy level to keep",
+            }),
+            outputFile: option({
+                type: optional(string),
+                long: "output-file",
+                description: "Name of the output file that will contain the sql commands",
+            }),
+            overwrite: flag({
+                long: "overwrite",
+                description: "Overwrite ouput file if exists",
+            }),
+        },
+        handler: async args => {
+            const outputFile = args.outputFile ?? "remove_orgunits.sql";
+            const sqlCommands = removeOrgs(`hierarchylevel > ${args.level}`);
+            safeWrite(outputFile, sqlCommands, args.overwrite);
+            process.exit(0);
+        },
+    });
+
+    const removeOrgUnitChildrenCommand = command({
+        name: "remove-orgunit-children",
+        description: "Create a sql file that removes all children of an organisation unit",
+        args: {
+            path: positional({
+                type: OrgUnitPath,
+                description: "Path of the organisation unit to prune of its children",
+            }),
+            outputFile: option({
+                type: optional(string),
+                long: "output-file",
+                description: "Name of the output file that will contain the sql commands",
+            }),
+            overwrite: flag({
+                long: "overwrite",
+                description: "Overwrite ouput file if exists",
+            }),
+        },
+        handler: async args => {
+            const outputFile = args.outputFile ?? "remove_orgunits.sql";
+            const sqlCommands = removeOrgs(`path LIKE '${args.path}/%'`);
+            safeWrite(outputFile, sqlCommands, args.overwrite);
+            process.exit(0);
+        },
+    });
+
+    return subcommands({
+        name: "database",
+        cmds: {
+            "remove-orgunit-level": removeOrgUnitLevelCommand,
+            "remove-orgunit-children": removeOrgUnitChildrenCommand,
+        },
+    });
+}
+
+// Write to outputFile the given string str, unless the file exists.
+function safeWrite(outputFile: string, str: string, overwrite: boolean) {
+    if (!fs.existsSync(outputFile) || overwrite) {
+        fs.writeFileSync(outputFile, str);
+        console.log(`Written: ${outputFile}`);
+    } else {
+        console.error(`ERROR. Output file already exists: ${outputFile}`);
+        console.error("Use --overwrite if you want to overwrite it anyway.");
+    }
+}
+
+// Return an SQL string that would remove all the organisation units that
+// satisfy the given whereCondition.
+function removeOrgs(whereCondition: string): string {
+    return `
```

Comment:
No strong opinion, but I am a bit wary of big chunks of interpolated strings. If you prefer to keep it like this, no problem. The alternative is to create a separate: `somefile.template.sql`, read the contents, and interpolate some namespace (for example: using _.template)

Reply:
Yes, I like it that way too, I just didn't know about _.template, thanks! Not sure about its proper location, though. I added it to `./src/data/` in 5ced1758166c019f10c8572e861edeb175f2728c .